apps/timetable/admin/ui/css/admin.css

```diff
@@ -55,6 +55,17 @@
 }
 
 
+/***********/
+/* POPOVER */
+/***********/
+
+.popover {
```

Comment:
I think Bootstrap also uses the `.popover` class. Could you check that the popovers in the student UI are still showing up properly?